composer.json

```diff
@@ -4,20 +4,20 @@
     "type": "project",
     "license": "proprietary",
     "require": {
-        "php": "^8.0",
-        "dvsa/mot-cpms-notifications": "^1.0.0",
+        "php": "^8.2",
+        "dvsa/mot-cpms-notifications": "dev-feature-bl-17032-php-82-update",
```

Comment:
Create a tag in dependency repos and use it here. You can create a tag before merging to master, and after merging you can correct it if needed.

Reply:
As discussed with @aaronlong , we will create tags once the swimlane is ready